agent/lib/kontena/service_pods/creator.rb

```diff
@@ -159,6 +159,7 @@ def ensure_image(name)
       # @param [Docker::Container] service_container
       # @return [Boolean]
       def service_uptodate?(service_container)
+        return false unless service_container.running?
```

Comment:
Does it really matter? We don't really protect containers so why not swap to fresh one here?